pkgs/build-support/cc-wrapper/ld-wrapper.sh

```diff
@@ -121,7 +121,7 @@ if [ "$NIX_@infixSalt@_DONT_SET_RPATH" != 1 ]; then
     # It's important to add the rpath in the order of -L..., so
     # the link time chosen objects will be those of runtime linking.
     declare -A rpaths
-    for dir in "${libDirs[@]}"; do
+    for dir in ${libDirs+"${libDirs[@]}"}; do
```

Comment:
@edolstra we can get rid of this as soon as we bump the bootstrap tools to 4.4 bash. Unlike `setup.sh`, this doesn't ever run in system bash due to leaky `nix-shell`.